src/map/utils/battleutils.cpp

```diff
@@ -365,6 +365,12 @@ namespace battleutils
         // than the size of the array.
         auto maxLevel = static_cast<uint8>(g_SkillTable.size() - 1);
 
+        // max indexed value and level is capped at 99.
+        if (level > 99)
+        {
+            level = 99;
+        }
```

Comment:
What effect can this have on current existing mobs that are over lvl 99?

Reply:
Every single instance of `uint16 GetMaxSkill` has `mlvl > 99 ? 99 : mlvl` in it. I am not sure why this wasn't just put in this function initially. Maybe I am missing something on this.

Reply:
There's been a lack of info on mob stats above 99, so our predecessors band-aid'd with a good ol "we'll do this for now and fix properly it later". 

Even 76-99 was just repeating lv 74->75's stat gains for a long time.

This little check probably fixed a few out of bound stats I've seen happen around level 108 or so...

Reply:
> 

76 to 99 should have the correct stats - the table does scale to 99

Reply:
> My understanding was they are level 99 with enhanced stats, 
they for sure do that with players pets.

With mobs you fight they actually are reporting higher levels in the packet data, but my suspicion is mobs have 2 "kinds" of level - the actual level and then the level they _say_ they are. Kind of like players that are "119" are still actually level 99. I had suspected this before item level was a thing, because of detectors up in sky and the fact mobs report a level is 2 different packets. I figure when we got item levels they just extended a version of the same thing over to players.

> I can add a TODO: with this line for research on 99+ mobs which isn't implemented atm anyway.

sounds good to me